File: C#/RefactoringWorkshop/RefactoringWorkshop/Coupler/MessageChain/SubscriptionWrapper.cs
using System.Collections.Generic;

namespace RefactoringWorkshop.Coupler.MessageChain
{
    public class SubscriptionWrapper
    {
        public List<SubscriptionPreferenceWrapper> SubscriptionPreferences { get; set; } =
            new List<SubscriptionPreferenceWrapper>();

        public void AddSubscriptionPreferences(SubscriptionPreferenceWrapper subscriptionPreference) {
            SubscriptionPreferences.Add(subscriptionPreference);
        }

    }
}

File: Java/RefactoringWorkshop/src/main/java/coupler/messagechain/SubscriptionWrapper.java
package coupler.messagechain;

import java.util.ArrayList;
import java.util.List;

public class SubscriptionWrapper {
    private List<SubscriptionPreferenceWrapper> subscriptionPreferences = new ArrayList<>();

    public List<SubscriptionPreferenceWrapper> getSubscriptionPreferences() {
        return this.subscriptionPreferences;
    }

    public void addSubscriptionPreferences(SubscriptionPreferenceWrapper subscriptionPreference) {
        this.subscriptionPreferences.add(subscriptionPreference);
    }
}
